codegen/directive.go

```diff
@@ -19,9 +20,10 @@ func (dl DirectiveList) LocationDirectives(location string) DirectiveList {
 
 type Directive struct {
 	*ast.DirectiveDefinition
-	Name    string
-	Args    []*FieldArgument
-	Builtin bool
+	Name string
+	Args []*FieldArgument
+
+	config.DirectiveConfig
```

Comment:
There can be multiple config properties, I thought it was easier to just pass this through. I can change this to an explicit map if you'd prefer.